ci-operator/config/shiftstack/shiftstack-ci/shiftstack-shiftstack-ci-main__periodic-4.14.yaml

```diff
@@ -118,6 +118,15 @@ tests:
       ROTATE_CLOUD_CREDENTIALS: "true"
       TEST_SUITE: openshift/conformance/serial
     workflow: openshift-e2e-openstack-ipi
+- as: e2e-openstack-ovn-masterscale
+  minimum_interval: 240h
+  steps:
+    cluster_profile: openstack-vexxhost
```

Comment:
you'll probably want our provider to create the alternative flavor; I don't think we can do it ourselves.

Reply:
The request is open in their ticketing system already.

Reply:
and it's done! I am updating the conf with the name they chose.